devel/gmp/Portfile

```diff
@@ -142,8 +142,16 @@ if {${os.platform} eq "darwin" && ${os.major} < 10 && [string match *clang* ${co
     configure.args-append           lt_cv_path_NM=${prefix}/bin/nm
 }
 
+
+if {${build_arch} eq "arm64"} {
+    patchfiles-append arm64-darwin-support.patch
+    use_autoreconf    yes
```

Comment:
This won't work for universal builds made on an Intel Mac (where build_arch will still be x86_64). Also, it will be easy for maintainers using an Intel Mac to break this patch unintentionally because it doesn't apply on Intel systems. Unless this patch causes problems for Intel systems, I would apply it unconditionally.

Reply:
I don't think so; I'll change it so it's unconditional.